code/src/index.css

```diff
@@ -1,13 +1,137 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Fredoka', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  overflow-x: hidden;
+  background-color: #822BD9;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.app-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-end;
+  width: 75vw;
+}
+
+form {
+  margin-top: 20px;
+  margin-bottom: 40px;
+  align-self: center;
+  grid-area: form;
+}
+
+label {
+  font-size: 21px;
+}
+
+[type="text"]{
+  font-size: 17px;
+  padding: 12px;
+  border: 4px solid #D932AC;
+  border-radius: 5px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+  margin-top: 20px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  display: block;
+  width: 45vw;
+}
+
+[type="checkbox"]{
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  width: 30px;
+  height: 30px;
+  border: 3px solid #D932AC;
+  border-radius: 50%;
+  background-color: #fff;
+}
+[type="checkbox"]:checked {
+  border: 3px solid #D932AC;
+  background-color: #D9725B;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+}
+
+[type="button"]{
+  background-color: #D9725B;
+  border: none;
+  border-radius: 5px;
+  width: 30px;
+  height: 30px;
+}
+
+[type="button"]:hover {
+  background-color: #D932AC;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2); 
+}
+
+ul {
+  grid-area: list;
+}
+
+.list-item {
+  font-size: 20px;
+  border: 1px solid black;
+  padding: 15px 5px 15px 5px;
+  background-color: white;
+  display: grid;
+  grid-template-columns: 1fr 7fr 1fr;
+  align-items: center;
+  }
+
+.counter {
+  margin: 10px;
+  align-self: flex-end;
+  font-size: 21px;
+  grid-area: counter;
+}
+
+.lazy-cat-main {
+  opacity: 0;
+}
+
+@media (min-width: 1024px) {
+  .main {
+    display: grid;
+    grid-template-areas: 
+    "list form"
+    "counter form";
+    margin-bottom: 11vw;
+  }
+  [type="text"] {
+    width: 25vw;
+  }
+
+  form {
+    justify-self: flex-end;
+    margin-top: 0;
+    align-self: center;
+  }
+
+  .counter {
+    align-self: flex-start;
+    text-align: start;
+  }
+
+  .lazy-cat-header {
```

Comment:
interesting way of solving the moving of the animation between different screen sizes